compute/packages_test.go

```diff
@@ -11,12 +11,15 @@ package compute_test
 import (
 	"context"
 	"errors"
+	"fmt"
 	"io/ioutil"
 	"net/http"
 	"path"
 	"strings"
 	"testing"
 
+	"github.com/abdullin/seq"
```

Comment:
Where did you find this?  This looks like a very useful testing library.

Can you `dep ensure` and add `seq` to this PR?

Reply:
I can't take the credit, it came from James and the image API tests. But yes, it is handy.